language/move-model/src/ast.rs

```diff
@@ -961,6 +961,7 @@ pub enum Value {
     Bool(bool),
     ByteArray(Vec<u8>),
     AddressArray(Vec<BigUint>),
```

Comment:
Maybe leave a TODO item here to indicate that the `AddressArray` may be merged into `Vector` in the future?

Reply:
done